### ISSUE TITLE
Input coordinates clear understanding by pointer icons

### ISSUE THREAD

**Issue by aashna27** [publiclab/image-sequencer#1193]
### We need to make coordinate input fields more descriptive to let the user know that values can be modified by clicking on image coordinates.
Suggestions:

- use a cross hairs icon 
- use a mouse or finger pointer icon
- while click event is active, change the cursor itself.

This is an improvement for the current open pr, #1187 and ideas are sought for better inputs.

> What happened just before the problem occurred? Or what problem could this idea solve?

> What did you expect to see that you didn't?

### Please show us where to look

https://beta.sequencer.publiclab.org

### What's your PublicLab.org username?

> This can help us diagnose the issue:

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

**Comment by jywarren** [comment]
Great!! Do you think it should trigger the "choose a pixel" interface when
you click the icon? And, when adding the module, should it auto click it,
basically, to prompt the user initially?

**Comment by aashna27** [comment]
Maybe what we can do is that when the icon is clicked the image comes in focus.What say?

**Comment by MargaretAN9** [comment]
Being able to identify image  coordinates could also be used as part of  a module to calibrate microscope images. Once the points are identified a scaling routine could be sent to the grid module as in  https://publiclab.org/system/images/photos/000/025/299/original/Slide2.JPG

**Comment by vaibhavmatta** [comment]
<img width="913" alt="Screenshot 2019-08-10 at 1 21 56 AM" src="https://user-images.githubusercontent.com/50797619/62805329-48b06700-bb0d-11e9-89f0-eedee11906d7.png">
Can’t we give a CROP PHOTO button just below the Initial step and make a new node.js app for cropping it? It will look much professional and easier to use.

**Comment by jywarren** [comment]
> Maybe what we can do is that when the icon is clicked the image comes in focus.What say?

What does "comes in focus" look like? 

@vaibhavmatta sorry, i don't quite follow; this isn't specifically a crop issue, can you clarify, and if you'd like to open a new issue for a separate topic, please do!

**Comment by vaibhavmatta** [comment]
@jywarren I am talking about giving the option to crop the image. Should I open a new issue for it? I will design my own UI to address the issue. Isn't it good?

**Comment by jywarren** [comment]
ah, i think crop should now have this feature, shouldn't it? But, we should
try to imagine the exact UI that will be standard for pixel x,y selection.
I think ideally you click the icon next to the box, and you get a
crosshairs cursor, and then yes, we could somehow highlight the image to
prompt you to click on it?

**Comment by vaibhavmatta** [comment]
@jywarren What if the image gets highlighted as soon as User clicks Crop button and It would be easier for user if he just selects the important area out from picture instead of filling in the values of X,Y etc. .

**Comment by jywarren** [comment]
Ah, yes, so that would be the case upon initially adding the step, right?
Yes, i think that makes sense too.

**Comment by vaibhavmatta** [comment]
@jywarren exactly. May I Claim it? I think I can work on it. It's a new feature and I would love to work on it.

**Comment by jywarren** [comment]
Sure, if Aashna is ok with it? Thank you!

**Comment by vaibhavmatta** [comment]
@jywarren Welcome Sir. Please keep in mind, my Interest and love for Publiclab. You can tag me in any issue you want me to work on. I would have pleasure in Contributing. 
@aashna27 please confirm. Thanks.

**Comment by jywarren** [comment]
Great, thank you! If you are still waiting, there are plenty of other
issues you can take up as well! Have you done a first-timers-only issue
yet? Thanks!

**Comment by vaibhavmatta** [comment]
@jywarren Being bit experienced in OpenSource Contributions, I am not touching first-timers-only issues. They are meant to help first-timers to build their Confidence. 
Still if you say, I will go for them 🙂.

**Comment by jywarren** [comment]
well, you are welcome to do it in order to learn our particular workflow,
but I also appreciate your willingness to jump in on a more challenging
task! We also have "help-wanted" issues. Thanks a lot!

**Comment by vaibhavmatta** [comment]
@jywarren, I will claim some of them 🙂.
